openhands/resolver/issue_definitions.py

```diff
@@ -1,515 +1,97 @@
-                my_review_threads = node.get('comments', {}).get('nodes', [])
-                message = ''
-                files = []
-                for i, review_thread in enumerate(my_review_threads):
-                    if (
-                        comment_id is not None
-                        and int(review_thread['fullDatabaseId']) == comment_id
-                    ):
-                        thread_contains_comment_id = True
-
-                    if (
-                        i == len(my_review_threads) - 1
-                    ):  # Check if it's the last thread in the thread
-                        if len(my_review_threads) > 1:
-                            message += '---\n'  # Add "---" before the last message if there's more than one thread
-                        message += 'latest feedback:\n' + review_thread['body'] + '\n'
-                    else:
-                        message += (
-                            review_thread['body'] + '\n'
-                        )  # Add each thread in a new line
-
-                    file = review_thread.get('path')
-                    if file and file not in files:
-                        files.append(file)
-
-                if comment_id is None or thread_contains_comment_id:
-                    unresolved_thread = ReviewThread(comment=message, files=files)
-                    review_threads.append(unresolved_thread)
-                    thread_ids.append(id)
-
-        return (
-            closing_issues_bodies,
-            closing_issue_numbers,
-            review_bodies,
-            review_threads,
-            thread_ids,
-        )
+        issues_context = json.dumps(issue.closing_issues, indent=4)
+        success_list = []
+        explanation_list = []
 
-    # Override processing of downloaded issues
-    def _get_pr_comments(
-        self, pr_number: int, comment_id: int | None = None
-    ) -> list[str] | None:
-        """Download comments for a specific pull request from Github."""
-        url = f'https://api.github.com/repos/{self.owner}/{self.repo}/issues/{pr_number}/comments'
-        headers = {
-            'Authorization': f'token {self.token}',
-            'Accept': 'application/vnd.github.v3+json',
-        }
-        params = {'per_page': 100, 'page': 1}
-        all_comments = []
-
-        while True:
-            response = requests.get(url, headers=headers, params=params)
-            response.raise_for_status()
-            comments = response.json()
-
-            if not comments:
-                break
-
-            if comment_id is not None:
-                matching_comment = next(
-                    (
-                        comment['body']
-                        for comment in comments
-                        if comment['id'] == comment_id
-                    ),
-                    None,
+        # Handle PRs with file-specific review comments
+        if issue.review_threads:
```

Comment:
In Github, we have `review_threads`, `thread_comments`, and `review_comments`

Is this same in Gitlab? Reason I ask, is because I'd imagine the next step from this PR is to create a `GitlabHandler` which can be plugged right into `ServiceContext`

If Github and Gitlab don't match on this terminology, we may need to do further abstracting on the `IssueHandler` level? LMK what you think!